File: app/src/main/scala/Demo/CSElement/Wrapper.scala
package Demo.CSElement

import wcs.scala._
import wcs.tag._

import java.io.CharArrayWriter
import java.io.PrintWriter
import Demo.test._

class Wrapper extends Element with Log {

  def apply(e: Env) = Seq(
    <h1>Hello</h1>,
    "<p><b>",
    call("HelloWorld"),
    "</b></p>").toString

}

File: app/src/main/java/agilesites/typeless/Wrapper.java
package agilesites.typeless;

import wcs.java.Asset;
import wcs.java.Element;
import wcs.java.Env;

public class Wrapper extends Element {

	@Override
	public String apply(Env e) {
		Asset a = e.getAsset(e.getC(), e.getCid());
		return a.call(a.getTemplate());
	}

}
